gnssanalysis/gn_io/sp3.py

```diff
@@ -191,6 +191,23 @@ def sp3_clock_nodata_to_nan(sp3_df: _pd.DataFrame) -> None:
     sp3_df.loc[nan_mask, ("EST", "CLK")] = _np.nan
 
 
+def remove_offline_sats(sp3_df: _pd.DataFrame):
```

Comment:
Can we have an unit test on this one to make sure it works as expected?
Needs maybe 2 epochs, 2 satellite one with `0`.